packages/samples/test/output/grpc-library-example/openapi.json

```diff
@@ -185,7 +185,17 @@
         },
         "tags": [
           "LibraryService"
-        ]
+        ],
+        "requestBody": {
```

Comment:
wow, how did we miss this?

Reply:
probably happened during the migration to interface and there was too much noise to notice